frontend/components/Dashboard/Nav/Nav.tsx

```diff
@@ -69,15 +68,15 @@ const Nav: React.FC<Props> = ({ currentUser, expanded, collapse }) => {
           position: fixed;
           top: 0;
           left: 0;
-          display: flex;
-          flex-direction: column;
+          display: grid;
```

Comment:
The _only_ way I was able to get the nav links centered on the screen was to switch to `grid`. Apparently you can't do `justify-self` with `flex` when you're using `flex-direction: column` 😞